pydm/data_plugins/epics_plugins/pyepics_plugin_component.py

```diff
@@ -47,6 +45,13 @@ def __init__(self, channel, pv, protocol=None, parent=None):
         self._upper_warning_limit = None
         self._lower_warning_limit = None
 
+        thread = CAThread(target=self.setup_callbacks, args=(channel,))
```

Comment:
```suggestion
        thread = CAThread(target=self.setup_callbacks, args=(channel,), daemon=True)
```
`daemon=True` to not let the thread running stop us from quitting pydm